python/plugins/db_manager/dlg_export_vector.py

```diff
@@ -163,7 +163,7 @@ def accept(self):
             if self.chkDropTable.isChecked():
                 options['overwrite'] = True
 
-            outCrs = None
+            outCrs = QgsCoordinateReferenceSystem
```

Comment:
Not required here; @nyalldawson , should there be one as per an established rule?